src/Arcus.Messaging.Pumps.EventHubs/AzureEventHubsMessagePump.cs

```diff
@@ -55,13 +55,20 @@ internal class AzureEventHubsMessagePump : MessagePump
 
             _eventHubsConfig = eventHubsConfiguration;
             _messageRouter = messageRouter;
-            _loggingScope = logger.BeginScope("Job: {JobId}", Id);
+            
+            JobId = _eventHubsConfig.Options.JobId;
+            _loggingScope = logger.BeginScope("Job: {JobId}", JobId);
         }
 
         private string EventHubName => _eventProcessor?.EventHubName;
         private string ConsumerGroup => _eventProcessor?.ConsumerGroup;
         private string Namespace => _eventProcessor?.FullyQualifiedNamespace;
 
+        /// <summary>
+        /// Gets the unique message pump ID to identify the pump that needs to be restarted.
+        /// </summary>
+        public string JobId { get; }
```

Comment:
Should we use `<inheritdoc/>` in the comments section ?
(For all implemented interface members ? )

Reply:
https://github.com/arcus-azure/arcus.messaging/pull/338/commits/49b8bea3e8ab54c2f1e4a57a76fa0a85f4803bb7